components/VC/Views/VCCardViewContent.tsx

```diff
@@ -90,6 +90,7 @@ export const VCCardViewContent: React.FC<VCItemContentProps> = props => {
               <VCVerification
                 wellknown={props.wellknown}
                 isVerified={props.isVerified}
+                isExpired={props.isExpired}
```

Comment:
can we get this `isExpired` field from  `vcMetadata` directly if available?

Reply:
Done